### PR TITLE
Issues

### PR DESCRIPTION
**Pull request by yqin43**
## Description
solve the issues
#30  removed hard coded path to PyMOAB
#32 pep8 issues
#62 updated jupyter notebook
#52 add parameters to t_p_v function to determine whether to ignore the isolated vertices

### CODE REVIEW

**Comment by gonuke** [comment]
This probably needs to be rebased against `main`

**Comment by kkiesling** [comment]
> This probably needs to be rebased against `main`

impeccable timing @gonuke

File: Matplotlib.ipynb [hunk 1]
```diff
@@ -235,21 +319,21 @@
...
+    "version": 2
    },
    "file_extension": ".py",
    "mimetype": "text/x-python",
    "name": "python",
    "nbconvert_exporter": "python",
-   "pygments_lexer": "ipython3",
-   "version": "3.6.7"
+   "pygments_lexer": "ipython2",
+   "version": "2.7.16"
```
Comment (kkiesling) [inline Matplotlib.ipynb:336]: @gonuke - do you think it matters that this is stored as python 2 vs 3? We had some kernel issues with running in python 3.

File: dagmc_stats/dagmc_stats.py [hunk 2]
```diff
@@ -104,6 +104,8 @@ def get_triangles_per_vertex(my_core, native_ranges):
     my_core : a MOAB Core instance
     native_ranges : a dictionary containing ranges for each native type in the
                     file (VERTEX, TRIANGLE, ENTITYSET)
+    ignore_zero : a boolean value that indicates whether to ignore the zero
+                  values or not
```
Comment (kkiesling) [inline dagmc_stats/dagmc_stats.py:108]: `to ignore the vertices that are not connected to any triangles.`

File: Matplotlib.ipynb (hunk 1 above)
Comment (gonuke) [reply Matplotlib.ipynb:336]: It's a little concerning that it's not working in Py3.  Do you know what the issues are?

Comment (kkiesling) [reply Matplotlib.ipynb:336]: This was I think an issue with @yqin43 's system (couldn't get matplotlib to work or something like that in the notebook I think). It wasn't an issue w/ DAGMC stats.

Comment (gonuke) [reply Matplotlib.ipynb:336]: It's probably not the end of the world...

Comment (kkiesling) [reply Matplotlib.ipynb:336]: we could probably just manually edit the file back to python 3 in these lines..

Comment (kkiesling) [inline Matplotlib.ipynb:336]:
```suggestion
   "pygments_lexer": "ipython3",
   "version": "3.6.7"
```

File: dagmc_stats/dagmc_stats.py (hunk 2 above)
Comment (kkiesling) [inline dagmc_stats/dagmc_stats.py:108]:
```suggestion
    ignore_zero : a boolean value that indicates whether to ignore the 
                          vertices that are not connected to any triangles.
```

File: dagmc_stats/dagmc_stats.py [hunk 3]
```diff
@@ -104,6 +104,8 @@ def get_triangles_per_vertex(my_core, native_ranges):
     my_core : a MOAB Core instance
     native_ranges : a dictionary containing ranges for each native type in the
                     file (VERTEX, TRIANGLE, ENTITYSET)
+    ignore_zero : a boolean value that indicates whether to ignore the 
+           vertices that are not connected to any triangles.
```
Comment (yqin43) [inline dagmc_stats/dagmc_stats.py:108]:
```suggestion
                  vertices that are not connected to any triangles.
```

**Review by kkiesling — APPROVED** [review]
thanks for these fixes!

**Pull request merged**